README.md

```diff
@@ -1,602 +1,387 @@
 
-### Upgrade note
+#### Feature specs
 
-`route_for` from rspec-rails-1.x is gone. Use `route_to` and `be_routable`
-instead.
+Feature specs serve the same purpose,
+but `SystemTestCase` solves some longstanding configuration issues they had,
+so the RSpec team [officially recommends system specs][] over feature specs.
 
-## Helper specs
-
-Helper specs default to residing in the `spec/helpers` folder. Tagging any
-context with the metadata `:type => :helper` treats its examples as helper
-specs.
-
-Helper specs mix in ActionView::TestCase::Behavior. A `helper` object is
-provided which mixes in the helper module being spec'd, along with
-`ApplicationHelper` (if present).
+If you don’t have the luxury of upgrading,
+you’ll be limited to writing feature specs for your browser tests.
+They require [Capybara][],
+so make sure it’s listed in the `:test` group of your `Gemfile`:
 
 ```ruby
-require 'rails_helper'
-
-RSpec.describe EventsHelper, :type => :helper do
-  describe "#link_to_event" do
-    it "displays the title, and formatted date" do
-      event = Event.new("Ruby Kaigi", Date.new(2010, 8, 27))
-      # helper is an instance of ActionView::Base configured with the
-      # EventsHelper and all of Rails' built-in helpers
-      expect(helper.link_to_event).to match /Ruby Kaigi, 27 Aug, 2010/
-    end
-  end
+group :test do
+  gem "capybara"
 end
 ```
 
-## Matchers
-
-Several domain-specific matchers are provided to each of the example group
-types. Most simply delegate to their equivalent Rails' assertions.
-
-### `be_a_new`
-
-- Available in all specs
-- Primarily intended for controller specs
-
-```ruby
-expect(object).to be_a_new(Widget)
-```
-
-Passes if the object is a `Widget` and returns true for `new_record?`
-
-### `render_template`
-
-- Delegates to Rails' `assert_template`
-- Available in request, controller, and view specs
-
-In request and controller specs, apply to the `response` object:
-
-```ruby
-expect(response).to render_template("new")
-```
-
-In view specs, apply to the `view` object:
-
-```ruby
-expect(view).to render_template(:partial => "_form", :locals => { :widget => widget } )
-```
-
-### `redirect_to`
-
-- Delegates to `assert_redirect`
-- Available in request and controller specs
-
-```ruby
-expect(response).to redirect_to(widgets_path)
-```
-
-### `route_to`
+#### Request specs
 
-- Delegates to Rails' `assert_routing`
-- Available in routing and controller specs
+Request specs are for testing the application
+from the perspective of a _machine client._
+They begin with an HTTP request and end with the HTTP response,
+so they’re faster than feature specs,
+but do not examine your app’s UI or JavaScript.
 
-```ruby
-expect(:get => "/widgets").to route_to(:controller => "widgets", :action => "index")
-```
+Request specs provide a high-level alternative to unit testing controllers.
+In fact, as of RSpec 3.5,
+the RSpec team [discourages controller specs entirely][]
+in favor of request specs.
```

Comment:
The Rails team makes this recommendation also.

Reply:
Added.